test/cypress/features/wallet.feature

```diff
@@ -6,11 +6,6 @@ Feature: Wallet
     Then I should see 30 transactions
     When I click show more
     Then I should see more than 30 transactions
-    Then I click filter incoming
-    Then I should see incoming transaction in table
```

Comment:
That should be working right?

Reply:
As far as I remember, we have removed those filter buttons.

Reply:
yes, these tabs are removed.